contrib/pg_tde/documentation/docs/yum.md

```diff
@@ -4,6 +4,24 @@ This tutorial shows how to install `pg_tde` with [Percona Distribution for Postg
 
 Check the [list of supported platforms](install.md#__tabbed_1_1).
 
+## Memory limits for `pg_tde` keys
+
+A memory lock (`mlock`) is the maximum amount of RAM a process can occupy. This value differs between operating systems. You can check the current setting with this command:
```

Comment:
Technically, mlock is a syscall to lock memory and the limit is RLIMIT_MEMLOCK. And as Zsolt noted, it is the limit on how much memory can be locked (locked meaning it's going to stay in the RAM and can't be swapped to the disk)